src/routes/auth.py

```diff
@@ -82,13 +88,17 @@ async def refresh_token(credentials: HTTPAuthorizationCredentials = Security(oau
     return {"access_token": access_token, "refresh_token": refresh_token, "token_type": "bearer"}
 
 
-@router.post("/auth/logout", status_code=status.HTTP_200_OK)
+@router.post("/logout", status_code=status.HTTP_200_OK)
 async def logout(token: str = Depends(oauth2_scheme), db: Session = Depends(get_db)):
     """
-    User logout and the addition to the black list of tokens
+    User logout and the addition into the black list of tokens
     """
-    await add_token_to_blacklist(token, db)
-    return {"detail": "Logged out successfully"}
+    try:
+        payload = jwt.decode(token, settings.jwt_secret_key, algorithms=[settings.algorithm])
+        await add_token_to_blacklist(token, db)
+        return {"detail": f"User with token {token} has logged out successfully."}
```

Comment:
Is displaying the token in the "detail" message secure and useful?

Reply:
agree. showed in in the tests and in our demo- moved out from the message